integrations/mysql/documentation.yaml

```diff
@@ -27,15 +37,17 @@ config_mods: |
       spec:
         containers:
   +     - name: exporter
-  +       image: prom/mysqld-exporter:v0.14.0
-  +       env:
-  +         - name: DATA_SOURCE_NAME
-  +           value: root:password@(localhost:3306)/
+  +       image: prom/mysqld-exporter:v0.15.0
+  +       args:
+  +         - --config.my-cnf=/home/my.cnf
   +       ports:
   +       - containerPort: 9104
   +         name: prometheus
-        - image: mysql:5.7
-          name: mysql
+  +       volumeMounts:
+  +       - mountPath: /home/my.cnf
+  +         subPath: my.cnf
+  +         name: mysql-exporter-config
+        - name: mysql
```

Comment:
Let's include `        - image: mysql:5.7` here

This is presented as a diff, so it's not super crucial and not meant to be operational JSON, but still good to document which sql version we tested against